.version

```diff
@@ -0,0 +1 @@
+0.5.0
```

Comment:
Not easier just to hard code this in the `__init__.py`?

Reply:
this is part of a githoo. `.version` is made in the git hook from the poetry version

Reply:
Yes, but could't this be done to just auto generate the __init__ file?